### ISSUE TITLE
dm integration?

### ISSUE THREAD

**Issue by krlmlr** [beanumber/etl#51]
The [new dm package](https://krlmlr.github.io/dm/) offers a class that holds a number of tables and their referential relationships, among other things. Would this be a good target container for {etl}?

**Comment by beanumber** [comment]
Yes, definitely! The support for keys and diagrams is very cool!

I'll keep an eye on [this...](https://github.com/krlmlr/dm)

Thanks @krlmlr !
